learners/setup.md

```diff
@@ -17,7 +18,13 @@ least a day in advance of the workshop.  If you encounter problems with the
 installation procedure, ask your workshop organizers via e-mail for assistance so
 you are ready to go as soon as the workshop begins.
 
-### Windows - [Video tutorial][video-windows]
+:::::::::::::::::::::::::::::::::::::::::::::::::::
+
+:::::::::::::::: solution
+
+### Windows
+
+Checkout the [video tutorial][video-windows] or:
```

Comment:
These numbered lists are looking a bit awkward, could you format it differently? Either by putting the text on the same line as the number or making the number a little header

Reply:
Well-spotted! I fixed it :)